### ISSUE TITLE
"Getting started" guide not working (no client id/password provided)

### ISSUE THREAD

**Issue by asmith26** [CarperAI/cheese#48]
I'm following https://cheese1.readthedocs.io/en/latest/started.html, but when I run the python command (related: https://github.com/CarperAI/cheese/pull/47) I get a URL but **no client id/password**:

```bash
python -m examples.image_selection
100%|██████ 1/1 [00:04<00:00,  4.97s/it]
~/miniconda3/envs/cheese/lib/python3.11/site-packages/gradio/components.py:206: UserWarning: 'rounded' styling is no longer supported. To round adjacent components together, place them in a Column(variant='box').
  warnings.warn(
~/miniconda3/envs/cheese/lib/python3.11/site-packages/gradio/components.py:224: UserWarning: 'border' styling is no longer supported. To place adjacent components in a shared border, place them in a Column(variant='box').
  warnings.warn(
Running on local URL:  http://127.0.0.1:7860
```

**Desktop:**
 - OS: Ubuntu
 - Browser: Firefox

Many thanks for any help! :)

**Comment by LouisCastricato** [comment]
@shahbuland

**Comment by shahbuland** [comment]
You need to create a client. Create client is what gives you a user/pass, not launch.

**Comment by ayulockin** [comment]
Hey @shahbuland I am also stuck on this page:
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/31141479/210102612-b3c85c42-acb6-4d55-a0b8-30af589261c6.png">

**Comment by ayulockin** [comment]
Okay I tried the `docs_example.py` example. 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/31141479/210103466-3e4a7a77-b484-45ee-973c-c88091becca4.png">

Turns out `(1, 673696)` is the (user id, password).

**Comment by asmith26** [comment]
Possibly another idea/quick question: is there a way to disable user authentication?

**Comment by ayulockin** [comment]
Oh nice. Thanks @asmith26 :)

**Comment by shahbuland** [comment]
> Possibly another idea/quick question: is there a way to disable user authentication?
> 
> UPDATE/FOUND FIX: Removed `share = True`, from these lines:
> 
> https://github.com/CarperAI/cheese/blob/5148e80ee5850cb80cc9eb23c934d703fdf2502b/cheese/client/__init__.py#L300
> 
> https://github.com/CarperAI/cheese/blob/5148e80ee5850cb80cc9eb23c934d703fdf2502b/cheese/client/gradio_client.py#L276

Bit confused by this. Share should just change whether the gradio demo is accessible by people outside your local connection. By disabling user auth, you are referring to the field where it asks for a user and password at start right? What would the desired behavior be (cc @ayulockin), for it to just get you to the labelling screen immediately without any identifying information being needed? I see two options:  
1. Add option to drop user info entirely so that user ID is never kept/needed for anything. All you'd have to press is a start button to get started.  
2. Add option so that you do have to enter a user ID, but no password.   
During development, I imagined that this imposed a security risk as it means anyone with url can start writing to your dataset.
What is preferred?

**Comment by asmith26** [comment]
> Bit confused by this. Share should just change whether the gradio demo is accessible by people outside your local connection. 

So I've found by removing `share=True`  has fixed the actual problem in this issue (i.e. that I couldn't run the "Getting started" guide). I think what was happening without this change is that gradio would try to create such a public link/app, but I think a proxy network was blocking this, hence the app would just hang. 

Maybe we could make this more customisable when [`cheese.launch()`](https://github.com/CarperAI/cheese/blob/3207719374797bcc2e094e36af06776aa8daa607/examples/image_selection.py#L162) is ran (my personal preference/thought is to have `share=False` the default and then a user can use something like `cheese.launch(share=True)` as required).

>By disabling user auth, you are referring to the field where it asks for a user and password at start right? 

This was just an idea to help me get the "Getting started" example running - I'm less worried about this now that I have fixed my issue, but it might be beneficial to remove user auth if it's possible and simplify things etc.
